tox.ini

```diff
@@ -67,6 +67,8 @@ setenv =
 skip_install = true
 deps =
     black==21.12b0
+    # The 8.1.0 release of click breaks black; so we pin.
```

Comment:
Not related except that black started breaking due to: https://github.com/psf/black/issues/2964